APIs_Drivers/LowPowerTimeout-example/main.cpp

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
could you rename the folder to the example?
something like `lowpowerTimeout_ex1` ?